app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -29,4 +29,31 @@ public function index()
 
         return view('backend.categories.index', ['categories' => $categories]);
     }
+
+    /**
+     * Delete a category and return book to category default.
+     *
+     * @param Category $category object category
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy(Category $category)
```

Comment:
chổ này câu lệnh `if` anh nghĩ em nên để cho validate xử lý tì ok hơn, tạo ra một request và validate `notIn`code sẽ clear hơn đó.
https://laravel.com/docs/5.5/validation#rule-not-in